testsuite/conformance-test-plugins/azure-arc-platform.yaml

```diff
@@ -1,7 +1,7 @@
 podSpec:
```

Comment:
This change is already present in the target branch. You can remove this change

Reply:
taken care